src/indentBlock.ml

```diff
@@ -713,6 +714,7 @@ let rec update_path config block stream tok =
     | [] -> []
     | h::p as path ->
         match kind with
+        | _ when config.i_align_params = Never -> path
```

Comment:
style preference, but I'd much prefer an `if ... else` above rather than a `_ when`

Reply:
I thought it was consistent with the `KParen` pattern, changed.